src/components/CurrentCityForecast/components/HourlyForecastChart/HourlyForecastChart.jsx

```diff
@@ -9,9 +9,15 @@ function HourlyForecastChart({ locationId }) {
   const [hours, temperatures] = prepareChartData(hourlyForecast);
 
   useEffect(async () => {
-    const { data } = await ApiService.getHourlyForecast(locationId);
-    setHourlyForecast(data.forecast);
-  }, []);
+    try {
+      if (locationId) {
+        const { data } = await ApiService.getHourlyForecast(locationId);
```

Comment:
Again, start moving this outside. Not sure if you were going to do that in this PR or in another